# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../taper/install_man.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- taper-6.9b/Makefile.orig	2005-09-06 12:58:10.000000000 -0700
+++ taper-6.9b/Makefile	2005-09-06 12:58:33.000000000 -0700
@@ -83,7 +83,7 @@
 	install --mode 755 --strip taper $(BINDIR) 
 	install --mode 755 --strip bg_backup $(BINDIR)
 	install --mode 755 --strip bg_restore $(BINDIR)
-	install --mode 644 docs/taper.8 $(MANDIR)
+	install -D --mode 644 docs/taper.8 $(MANDIR)/taper.8
 	ln -sf $(BINDIR)/taper $(ALTBIN)/taper
 	ln -sf $(BINDIR)/bg_backup $(ALTBIN)/bg_backup
 	ln -sf $(BINDIR)/bg_restore $(ALTBIN)/bg_restore
